Jenkinsfile

```diff
@@ -700,6 +700,7 @@ pipeline {
                 stage('win-RS1') {
                     when {
                         beforeAgent true
+                        not { changeRequest() }
```

Comment:
👍 guess we should update the other `master` selectors the same.

Reply:
Actually; just thinking of a slightly improved approach;

change `params.windowsRS1` default to `false`, and then do this;

```groovy
or {
  not { changeRequest() }
  expression { params.windowsRS1 }
}
```

If that works;

- RS1 (and, e.g. `ppc64le/s390x-integration-cli`) only run on non-PR jobs ("master")
- Unless someone ticks the `windowsRS1` checkbox

That way, we can easily run these tests on a specific PR if we see a need